File: jamcatch/jamcatch-assets/src/testEndToEnd/java/org/example/jamcatch/assets/end2end/JamCatchAssetsEnd2EndTest.java
package org.example.jamcatch.assets.end2end;

import static org.junit.jupiter.api.Assertions.assertTrue;

import org.example.javarca.engine.test.fixtures.RendererFixture;
import org.example.javarca.engine.test.fixtures.Screenshot;
import org.junit.jupiter.api.Test;

public class JamCatchAssetsEnd2EndTest {

    @Test
    public void test() {
        Screenshot result = RendererFixture.launchAndWaitForFinish();

        assertTrue(result.file().exists(), "Screenshot should exist");
    }
}

File: jamcatch/jamcatch-assets/src/testEndToEnd/kotlin/org/example/jamcatch/assets/end2end/JamCatchAssetsEnd2EndTest.kt
package org.example.jamcatch.assets.end2end

import org.example.javarca.engine.test.fixtures.RendererFixture
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class JamCatchAssetsEnd2EndTest {
    @Test
    fun test() {
        val result = RendererFixture.launchAndWaitForFinish()

        Assertions.assertTrue(result.file.exists(), "Screenshot should exist")
    }
}
